template/workflows/manifests/.github/workflows/azure-kubernetes-service.yml

```diff
@@ -42,11 +42,13 @@ env:
   AZURE_CONTAINER_REGISTRY: {{ .Config.GetVariableValue "AZURECONTAINERREGISTRY" }}
   CONTAINER_NAME: {{ .Config.GetVariableValue "CONTAINERNAME" }}
   CLUSTER_RESOURCE_GROUP: {{ .Config.GetVariableValue "CLUSTERRESOURCEGROUP" }}
-  CLUSTER_NAME: {{ .Config.GetVariableValue "CLUSTERNAME" }}
   DEPLOYMENT_MANIFEST_PATH: {{ .Config.GetVariableValue "DEPLOYMENTMANIFESTPATH" }}
   DOCKER_FILE: {{ .Config.GetVariableValue "DOCKERFILE" }}
   BUILD_CONTEXT_PATH: {{ .Config.GetVariableValue "BUILDCONTEXTPATH" }}
   NAMESPACE: {{ .Config.GetVariableValue "NAMESPACE" }}
+  RESOURCE_TYPE: Microsoft.ContainerService/{{ .Config.GetVariableValue "RESOURCETYPE" }}
```

Comment:
as this resource is always a cluster, it may make sense to call it "cluster resource type" to follow the convention used in this template